src/mouse/actions.js

```diff
@@ -19,13 +26,65 @@ function setMousedown() {
 
 function unsetMousedown() {
   return {
-   type: UNSET_MOUSEDOWN
+   type: UNSET_MOUSEDOWN,
+   
+ }
+}
+function setCordinate(coordinate) {
+  return {
+   type: SET_COORDINATE,
+   coordinate
+
+ }
+}
+
+function unsetCordinate() {
+  return {
+   type: UNSET_COORDINATE
+ }
+}
+function setSize(size) {
+  return {
+   type: SET_SIZE,
+   size
+
  }
 }
 
+function unsetSize() {
+  return {
+   type: UNSET_SIZE
+ }
+}
+function setGoals(goalsAdresses) {
+  return {
+   type: SET_GOALS,
+   goalsAdresses
+
+ }
+}
+
+function unsetGoals() {
+  return {
+   type: UNSET_GOALS }
+}
+
+
 export {
   SET_MOUSEDOWN,
   UNSET_MOUSEDOWN,
+  SET_COORDINATE,
+  UNSET_COORDINATE,
+  SET_SIZE,
+  UNSET_SIZE,
+  SET_GOALS,
+  UNSET_GOALS,
   setMousedown,
-  unsetMousedown
+  unsetMousedown,
+  setCordinate,
+  unsetCordinate,
```

Comment:
this function name is misspelled, replace with `setCoordinate` and `unsetCoordinate`